code/ai/ai_profiles.cpp

```diff
@@ -58,6 +58,26 @@ int ai_path_type_match(char *p)
 	return -1;
 }
 
+
+const char* AI_secondary_range_select_modes[] = {
+	"retail",
+	"aware",
```

Comment:
Change this to `range_aware`

Reply:
You marked this as resolved, but you didn't write any comments, nor do I see that the string has been changed.

Reply:
Anyway, the reason for my suggestion was future-proofing.  If anyone adds more options in the future, "aware" might become ambiguous, or less clear.  It was for the same reason that I suggested changing `AI_RANGE_SEC_SEL_MODE_AWARE` to `AI_RANGE_SEC_SEL_MODE_RANGE_AWARE`.

Reply:
I could have sworn it was originally AI_secondary_range_select_modes and I assumed you wanted more specificty in the early part of the name. I see no evidence that I actually made that change, but that misunderstanding is still why I made the changes I did. I hit resolve because I thought I had implemented that name change everywhere it applied, and had just somehow obscured it with git fumbling.

"aware" was intended to be future proof, representing the very basic level of range consideration vs the retail lack of any awareness. I guess arguably that could be a bit confusing in some cases, but I don't see how "range aware" is more clear and specific as a mode string for a setting named "range awareness" than "aware" by itself is.

Reply:
Ok.  I don't have a strong opinion on what the word should be, but I wanted to make sure future expansion possibilities were considered.  Also, I thought it was odd that the #define was changed but this wasn't.  (And if you aren't planning to change this string, you may want to consider changing the #define back.)

Reply:
Reviewing the name changes I did make now suggests I was considerably more tired than I realized when I made them. A push with more consistent names and a fix to the other issue is coming once my local build finishes, which takes five or ten minutes at least.